concent_api/core/message_handlers.py

```diff
@@ -1316,10 +1316,13 @@ def handle_send_subtask_results_verify(
             reason=message.concents.ServiceRefused.REASON.InvalidRequest,
         )
 
-    verification_deadline = calculate_additional_verification_call_time(
-        subtask_results_rejected.timestamp,
-        compute_task_def['deadline'],
-        task_to_compute.timestamp,
+    verification_deadline = (
```

Comment:
Wouldn't it better to have this equation extracted to a function (i.e. `calculate_verification_deadline) and unit-test it?

Reply:
Please consider that it is just a simple add of three elements. So it will be tested like `2 + 2 + 2 = 6`. Do you think it is necessary?

Reply:
maybe not necessary, but why not? ;) I mean if it is in a function, it's easier to substitute it (i.e. patch) and change if the need arises. And unit tests are always welcome ;)